deployment/helm/skaha/skaha-config/k8s-resources.json

```diff
@@ -0,0 +1,88 @@
+{
+  "cores-default-request": 1,
+  "cores-default-limit": 16,
+  "cores-default": 1,
+  "cores-default-headless": 1,
+  "cores-options": [
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9,
+    10,
+    11,
+    12,
+    13,
+    14,
+    15,
+    16
+  ],
+  "mem-gb-default-request": 4,
+  "mem-gb-default-limit": 192,
+  "mem-gb-default": 2,
+  "mem-gb-default-headless": 4,
+  "mem-gb-options": [
+    1,
+    2,
+    4,
+    6,
+    8,
+    10,
+    12,
+    14,
+    16,
+    20,
+    24,
+    26,
+    28,
+    30,
+    32,
+    36,
+    40,
+    44,
+    48,
+    56,
+    64,
+    80,
+    92,
+    112,
+    128,
+    140,
+    170,
+    192
+  ],
+  "gpus-options": [
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9,
+    10,
+    11,
+    12,
+    13,
+    14,
+    15,
+    16,
+    17,
+    18,
+    19,
+    20,
+    21,
+    22,
+    23,
+    24,
+    25,
+    26,
+    27,
+    28
+  ]
+}
```

Comment:
Thanks @aratikakadiya .  Just some restructuring of this JSON file, as we can organize the different categories:

```json
{
  "cores": {
    "default": 1,
    "default-request": 1,
    "default-limit": 16,
    "default-headless": 1,
    "options": [...]
  },
  "memory": {
    "default-request-gb": 4,
    "default-limit-gb": 192,
    "default-gb": 2,
    "default-headless-gb": 4,
    "options-gb": [...]
  },
  "gpus": {
    "options": [...]
  }
}
```